pipeline-scripts/buildlib.groovy

```diff
@@ -125,6 +125,15 @@ def initialize_openshift_dir() {
     echo "Initialized env.OPENSHIFT_DIR: ${env.OPENSHIFT_DIR}"
 }
 
+def init_artcd_working_dir() {
+    sh """
+    mkdir /tmp/artcd_working
```

Comment:
Would like to rename this to empty_dir or similar to indicate our intention here and that this is not meant to be a backup dir